### PR TITLE
Team: Add api endpoint to get details about a particular team.

### PR DESCRIPTION
**Pull request by TroJan**

### CODE REVIEW

File: apps/teams/views.py [hunk 1]
```diff
@@ -43,3 +43,24 @@ def team_list(request, challenge_pk):
...
+
+    try:
+        team = Team.objects.get(pk=pk)
+    except Team.DoesNotExist:
+        response_data = {'error': 'Team does not exist'}
+        return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
+
+    serializer = TeamChallengeSerializer(team)
+    response_data = serializer.data
+    return Response(response_data)
```
Comment (deshraj) [inline apps/teams/views.py]: We should pass the response status here, right?

File: apps/teams/views.py [hunk 2]
```diff
@@ -43,3 +43,24 @@ def team_list(request, challenge_pk):
...
             return Response(response_data, status.HTTP_201_CREATED)
         return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(['GET'])
+@permission_classes((permissions.IsAuthenticated,))
+@authentication_classes((TokenAuthentication,))
+def team_detail(request, challenge_pk, pk):
+    try:
+        challenge = Challenge.objects.get(pk=challenge_pk)      # noqa
```
Comment (TroJan) [inline apps/teams/views.py]: we run flake8 in travis. it will be error an there.

File: apps/teams/views.py (hunk 1 above)
Comment (TroJan) [inline apps/teams/views.py]: yeah, my bad.

File: apps/teams/views.py (hunk 2 above)
Comment (deshraj) [inline apps/teams/views.py]: No, autopep8 automatically modifies the code to follow the pep8 styling. Can you look at the 2nd point on this link https://github.com/Cloud-CV/EvalAI/blob/master/CONTRIBUTING.md

**Pull request merged**